packages/mirage-block-solo5/mirage-block-solo5.0.6.0/opam

```diff
@@ -0,0 +1,36 @@
+opam-version: "2.0"
+maintainer:   "martin@lucina.net"
+homepage:     "https://github.com/mirage/mirage-block-solo5"
+dev-repo:     "git+https://github.com/mirage/mirage-block-solo5.git"
+bug-reports:  "https://github.com/mirage/mirage-block-solo5/issues"
+authors:      ["Dan Williams" "Martin Lucina"]
+tags: [
+  "org:mirage"
+]
+
+build: [
+ ["dune" "subst"] {pinned}
+ ["dune" "build" "-p" name "-j" jobs]
+ ["dune" "runtest" "-p" name] {with-test}
+]
+
+depends: [
+  "ocaml" {>= "4.04.2"}
+  "dune"  {build & >= "1.0"}
```

Comment:
```suggestion
  "dune"  {>= "1.0"}
```

Reply:
@mseri Actually, I misread this. Why would this not be a build dependency only?

Reply:
dune is not forward compatible, so it needs to recompile all packages if you downgrade dune.  Making it `{build}` has never really worked, so we've removed them all from the main opam repository.

Reply:
> dune is not forward compatible ...
Ok, done in 73f88dd1de.